conftest.py

```diff
@@ -0,0 +1,104 @@
+import os
```

Comment:
I think you can use isort:
```suggestion
import os
from typing import (
    Iterator,
    TYPE_CHECKING
)

import boto3
import pytest
from moto import mock_dynamodb
from sqlalchemy import create_engine
from sqlalchemy.orm import sessionmaker

from src.connection.dal import DataAccessLayer, Base
```